src/diffusers/models/transformer_temporal.py

```diff
@@ -183,3 +183,153 @@ def forward(
             return (output,)
 
         return TransformerTemporalModelOutput(sample=output)
+
+
+class TransformerTemporalMotionModel(ModelMixin, ConfigMixin):
```

Comment:
Let's try to merge into Transformer2DModel instead

Reply:
Hmm I think it can be merged into the `TransformerTemporalModel`.